src/tableaus/sdirk_tableaus.jl

```diff
@@ -1827,3 +1827,244 @@ function SDIRK22Tableau(T)
+  a75 = convert(T,1335994250573//5691609445217)
+  a76 = convert(T,2273837961795//8368240463276)
+
+  bhat3 = convert(T,4469248916618//8635866897933)
+  bhat4 = -convert(T,621260224600//4094290005349)
+  bhat5 = convert(T,696572312987//2942599194819)
+  bhat6 = convert(T,1532940081127//5565293938103)
+  bhat7 = convert(T,2441//20000)
+  btilde3 = convert(T,BigInt(216367897668138065439709)//BigInt(153341716340757627089664345)) # bhat3-a73
+  btilde4 = -convert(T,BigInt(1719969231640509698414113)//BigInt(303097339249411872572263321)) # bhat4-a74
+  btilde5 = convert(T,BigInt(33321949854538424751892)//BigInt(16748125370719759490730723)) # bhat5-a75
+  btilde6 = convert(T,BigInt(4033362550194444079469)//BigInt(1083063207508329376479196)) # bhat6-a76
+  btilde7 = -convert(T,29//20000) # bhat7-γ
+
+  #c2 = 2γ
+  c3 = convert(T2,4276536705230//10142255878289)
+  c4 = convert(T2,67//200)
+  c5 = convert(T2,3//40)
+  c6 = convert(T2,7//10)
+
+
+  
+  #TODO():Convert These values for this Method
+  α21 = convert(T,2) # c2/γ
+  α31 = convert(T,42//125)
+  α32 = convert(T,83//125)
+  α41 = convert(T,-6//25)
+  α42 = convert(T,31//25)
+  α51 = convert(T,914470432//2064665255)
+  α52 = convert(T,798813//724780)
+  α53 = convert(T,-824765625//372971788)
+  α54 = convert(T,49640//29791)
+  α61 = convert(T,288521442795//954204491116)
+  α62 = convert(T,2224881//2566456)
+  α63 = convert(T,-1074821875//905317354)
+  α64 = convert(T,-3360875//8098936)
+  α65 = convert(T,7040//4913)
+
+  ea21 = convert(T,247//1000)
+  ea31 = convert(T,247//4000)
+  ea32 = convert(T,2694949928731//7487940209513)
+  ea41 = convert(T,464650059369//8764239774964)
+  ea42 = convert(T,878889893998//2444806327765)
+  ea43 = -convert(T,952945855348//12294611323341)
+  ea51 = convert(T,476636172619//8159180917465)
+  ea52 = -convert(T,1271469283451//7793814740893)
+  ea53 = -convert(T,859560642026//4356155882851)
+  ea54 = convert(T,1723805262919//4571918432560)
+  ea61 = convert(T,6338158500785//11769362343261)
+  ea62 = -convert(T,4970555480458//10924838743837)
+  ea63 = convert(T,3326578051521//2647936831840)
+  ea64 = -convert(T,880713585975//1841400956686)
+  ea65 = -convert(T,1428733748635//8843423958496)
+  ea71 = convert(T,760814592956//3276306540349)
+  ea72 = convert(T,760814592956//3276306540349)
+  ea73 = -convert(T,47223648122716//6934462133451)
+  ea74 = convert(T,71187472546993//9669769126921)
+  ea75 = -convert(T,13330509492149//9695768672337)
+  ea76 = convert(T,11565764226357//8513123442827)
+
+  #eb1 = 0
+  #eb2 = 0
+  eb3 = convert(T,9164257142617//17756377923965)
+  eb4 = -convert(T,10812980402763//74029279521829)
+  eb5 = convert(T,1335994250573//5691609445217)
+  eb6 = convert(T,2273837961795//8368240463276)
+  eb7 = convert(T,247//2000)
+
+  #ebtilde1 = 0
+  #ebtilde2 = 0
+  ebtilde3 = convert(T,BigInt(216367897668138065439709)//BigInt(153341716340757627089664345))
+  ebtilde4 = -convert(T,BigInt(1719969231640509698414113)//BigInt(303097339249411872572263321))
```

Comment:
we will want a dispatch that goes straight to Float64, since bigs will allocate if large enough, so surprisingly the startup time in these functions can get measurable.